src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,75 @@
+
+If you’re concerned about user privacy, you can exclude the `X-Forwarded-For HTTP` header, which is used to geolocate clients. If you exclude it, New Relic will geolocate your proxy as the client instead.
+
+### Recommendations for proxy URLs
+
+When configuring these properties, we recommend the following:
+
+* Use a first-party subdomain. Even though you can technically use a third-party subdomain, it’s likely to get blocked by ad blockers and security VPNs.
+* The subdomain URL pattern is `<domain>[:<port>][/<path>]`. Make sure to omit the protocol `https://` and ending URL slash. The `port` and `path` are optional and will be respected if added.
+* If you’re configuring both `assets` and  `beacon`, set different values for each proxy URL, such as using a different subdomain for each URL or one that contains a different path. This will make configuring the proxy server a lot easier.
+
+## Set up your proxy [#set-up]
+
+To set up your proxy, see the instructions below based on which deployment method you used to install the browser agent.
+
+### Copy/paste installation [#copy-paste]
+
+If you're using the copy/paste installation method, add the following configuration to your browser JavaScript snippet, making sure to add your URL to `assets` and/or `beacon`:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>,
+    proxy: {
+        assets: URL
+        beacon: URL
+    }
+}
+```
+
+
+### NPM installation [#npm]
+
+If you’re using the NPM browser installation method, add the following configuration to your browser agent, which will pass the `beacon` field in the `init` object:
+
+```js
+new BrowserAgent({ init: { proxy: {
+        beacon: <URL to send analytics>
+    }
+  }
+})
+```
+
+## Using proxies [#using-proxies]
+
+New Relic does not provide or support a cloud proxy service, so you must configure and manage your own proxy servers. To ensure full functionality, we make the following assumptions about your proxies:
+
+* **Load the browser agent code from the New Relic CDN and pass it back to the client.** If you’re using the `assets` field, your proxy should load the browser agent code from `https://js-agent.newrelic.com/` and pass it back to the client. For example, for browser agent v1.240.0, requests to `https://assets.mydomain.com/nr-spa.3b61d78f-1.240.0.min.js` should yield the same response as from the upstream `https://js-agent.newrelic.com/nr-spa.3b61d78f-1.240.0.min.js`.
+* **Forward client payloads to New Relic.** The `beacon` proxy forwards client payloads to New Relic. Your original endpoint can be found in your browser application settings page. Look for `;NREUM.info={beacon:"_____",...` in the JavaScript snippet. For most apps, this would be `(https://)bam.nr-data.net`. However, it could be different due to compliance and legal reasons. For example, requests to `https://yourproxy.com/1/asdf?a=123` should be passed through to `https://bam.nr-data.net/1/asdf?a=123`.
```

Comment:
```suggestion
* **Forward client payloads to New Relic.** The `beacon` proxy forwards client payloads to New Relic. Your original endpoint can be found in your [browser application settings page](/docs/browser/new-relic-browser/configuration/browser-app-settings-page/). Look for `;NREUM.info={beacon:"_____",...` in the JavaScript snippet. For most apps, this would be `(https://)bam.nr-data.net`. However, it could be different due to compliance and legal reasons. For example, requests to `https://yourproxy.com/1/asdf?a=123` should be passed through to `https://bam.nr-data.net/1/asdf?a=123`.
```